modules/qdrant/testcontainers/qdrant/__init__.py

```diff
@@ -0,0 +1,145 @@
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import os
+from functools import cached_property
+from pathlib import Path
+from typing import Optional
+
+from qdrant_client import AsyncQdrantClient, QdrantClient
```

Comment:
Can we please avoid coupling to the qdrant API?

Reply:
I added this based on the other container modules.
https://github.com/testcontainers/testcontainers-python/blob/5758310532b8a8e1303a24bc534fa8aeb0f75eb2/modules/mongodb/testcontainers/mongodb/__init__.py#L87

https://github.com/testcontainers/testcontainers-python/blob/5758310532b8a8e1303a24bc534fa8aeb0f75eb2/modules/neo4j/testcontainers/neo4j/__init__.py#L73

Reply:
I think we will circle back later to remove all of these - its too much to think about doing all up front. in the future i think we'll actually move the imports into the offending methods and surround with try except ImportError and re-raise with a message telling the user to install the module and not use the methods.

Reply:
Turned them to optional imports.

https://github.com/testcontainers/testcontainers-python/pull/463/commits/e9354f19577242ad6ee6907da2274edb3790ef43